src/components/WebexInMeeting/WebexInMeeting.scss

```diff
@@ -4,7 +4,8 @@ $shadow-spread: -0.125rem;
 $shadow-color: rgba(0, 0, 0, 0.2);
 
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-in-meeting {
-  background: black;
+  background: var(--wxc-in-meeting-background);
```

Comment:
only the meeting component has background

Reply:
Because this won't be needed after the release of the themes, let's just remove it from here and add a rule in legacy.scss:
```scss
.wxc-in-meeting { background: black; }
```
overridden to transparent in the temporary sections of the theme files. No need for css variables.